### ISSUE TITLE
:wheelchair: a11y(bal-date): Input Mask is read out by screenreader instead of label or placeholder

### ISSUE THREAD

**Issue by GinaBiondo** [baloise/design-system#1445]
### Description of this issue

When entering the input via screen reader the "format mask" is read out instead of the label or placeholder. 
<img width="358" alt="Bildschirmfoto 2024-09-03 um 17 12 53" src="https://github.com/user-attachments/assets/7f7328d0-a3fe-4aa9-8953-1d7b503ae5cc">

### How to fix it

Wen entering the input via screen reader the name / purpose of the input should be announced fist.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
Add aria label to make it explicit that this is masked date input

**Comment by hirsch88** [comment]
IBM will check again

**Comment by carolabes** [comment]
Currently the placeholder is recognized as value by screenreader. Therefore it is read as first element when entering date picker. The placeholder should not be treated as value but added as aria-describedby. 

Here is an example how it should work: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/

**Comment by hirsch88** [comment]
Cannot be done with a masked input.
